packages/popover/src/vaadin-popover.js

```diff
@@ -149,6 +149,18 @@ class PopoverOpenedStateController {
   }
 }
 
+/**
+ * Returns true if the popover overlay is the last one in the opened overlays stack, ignoring tooltips.
+ * @param {HTMLElement} overlay
+ * @return {boolean}
+ * @protected
+ */
+export const isLastOverlay = (overlay) => {
+  // Ignore tooltips, popovers should still close when a tooltip is present
+  const filter = (o) => o.constructor.is !== 'vaadin-tooltip-overlay';
```

Comment:
nit: I think we can check for localName instead:

```suggestion
  const filter = (o) => o.localName !== 'vaadin-tooltip-overlay';
```

Reply:
Done